packages/generate-pdf-template-helpers/src/helpers.ts

```diff
@@ -49,6 +49,11 @@ export function computeTextStyle(
   locales?: string | string[],
 ): string | string[];
 export function computeTextStyle(value: string, format?: PossibleFormat, locales?: string | string[]): string;
+export function computeTextStyle(
```

Comment:
This is the only fix I could find for the compiler issue here: https://cloud.nx.app/runs/MURvaFXPBf
The types are compiled by `json-schema-to-typescript`.
The problem type is `[string, ...string[]]`.
Is there a better way to solve this problem? @getlarge

Reply:
this are not really clean overloads...
Each overload should describe a specific input / output case plus one that contains all inputs / outputs unions.
```ts
// use this type for the function declaration and the most generic overload
type PossibleValues = string | string[] | [string, ...string[]];
export function computeTextStyle(
  value: PossibleValues,
  format?: PossibleFormat,
  locales?: string | string[],
): PossibleValues;

... all specific overloads in between

export function computeTextStyle(
  value: PossibleValues,
  format?: PossibleFormat,
  locales?: string | string[],
): PossibleValues {
...
}